Aspire Budgeting/Views/CardView/CardListView.swift

```diff
@@ -29,46 +33,34 @@ struct CardListView: View {
   }
 }
 
-extension CardListView {
-  static let colorInfos: [CardView.ColorInfo] =
-    [.init(gradientStartColor: .materialRed800,
-           gradientEndColor: .materialRed800,
-           shadowColor: .materialRed800),
-     .init(gradientStartColor: .materialPink800,
-           gradientEndColor: .materialPink800,
-           shadowColor: .materialPink800),
-     .init(gradientStartColor: .materialPurple800,
-           gradientEndColor: .materialPurple800,
-           shadowColor: .materialPurple800),
-     .init(gradientStartColor: .materialDeepPurple800,
-           gradientEndColor: .materialDeepPurple800,
-           shadowColor: .materialDeepPurple800),
-     .init(gradientStartColor: .materialIndigo800,
-           gradientEndColor: .materialIndigo800,
-           shadowColor: .materialIndigo800),
-     .init(gradientStartColor: .materialBlue800,
-           gradientEndColor: .materialBlue800,
-           shadowColor: .materialBlue800),
-     .init(gradientStartColor: .materialLightBlue800,
-           gradientEndColor: .materialLightBlue800,
-           shadowColor: .materialLightBlue800),
-     .init(gradientStartColor: .materialTeal800,
-           gradientEndColor: .materialTeal800,
-           shadowColor: .materialTeal800),
-     .init(gradientStartColor: .materialGreen800,
-           gradientEndColor: .materialGreen800,
-           shadowColor: .materialGreen800),
-     .init(gradientStartColor: .materialBrown800,
-           gradientEndColor: .materialBrown800,
-           shadowColor: .materialBrown800),
-     .init(gradientStartColor: .materialGrey800,
-           gradientEndColor: .materialGrey800,
-           shadowColor: .materialGrey800),
+//MARK:- Internal Types
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/5d9f74b3b35e3fb9746b1333ffd07f18323321a5/Aspire Budgeting/Views/CardView/CardListView.swift#L36"></span>MARK comment should be in valid format. e.g. ‘// MARK: …’ or ‘// MARK: - …’<br />
<code>mark</code> <code>CardListView.swift:36</code></td>
    </tr>
  </tbody>
</table>